pkg/didcomm/transport/http/inbound.go

```diff
@@ -111,7 +113,12 @@ func NewInbound(addr string) (*Inbound, error) {
 		return nil, errors.New("http address is mandatory")
 	}
 
-	return &Inbound{server: &http.Server{Addr: addr}}, nil
+	host, port, err := net.SplitHostPort(addr)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create new Inbound: %s", err)
+	}
+
+	return &Inbound{host: host, port: port, server: &http.Server{Addr: ":" + port}}, nil
```

Comment:
I do not understand why the address portion is being removed. This is used to bind to a particular interface.

Reply:
Addressed as discussed in the meeting earlier.